src/client/pages/Form.js

```diff
@@ -4,36 +4,57 @@ import React, {
 
 import BaseContainer from "@component/BaseContainer";
 import axios from "@utils/axios";
-import { isString } from "@utils/validate";
 
 export default (props) => {
-    const [japaneseName, setJapaneseName] = useState("");
-    const [englishName, setEnglishName] = useState("");
-    const [bio, setBio] = useState("");
+    // setup state as object
+    const [data, setData] = useState({
+        japaneseFirstName: "",
+        japaneseLastName: "",
+        englishFirstName: "",
+        englishLastName: "",
+        bio: ""
+    });
 
+    // set state in response to input name
     const handleChange = (e) => {
         switch (e.target.name) {
-            case "japanese_name":
-                setJapaneseName(e.target.value);
+            case "japanese_first_name":
+                setData(Object.assign(data, {
+                    "japaneseFirstName": e.target.value
+                }));
                 break;
-            case "english_name":
-                setEnglishName(e.target.value);
+            case "japanese_last_name":
+                setData(Object.assign(data, {
+                    "japaneseLastName": e.target.value
+                }));
+                break;
+            case "english_first_name":
+                setData(Object.assign(data, {
+                    "englishFirstName": e.target.value
+                }));
+                break;
+            case "english_last_name":
+                setData(Object.assign(data, {
+                    "englishLastName": e.target.value
+                }));
                 break;
             case "bio":
-                setBio(e.target.value);
+                setData(Object.assign(data, {
+                    "bio": e.target.value
+                }));
                 break;
             default:
                 break;
         }
     }
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        axios.post("/register", {
-            japaneseName: japaneseName,
-            englishName: englishName,
-            bio: bio
-        }).then(() => {
+    // validate and submit state data when button pushed
+    const handleSubmit = () => {
+        // trim and eliminate blank line in bio
+        axios.post("/register", Object.assign(data, {
```

Comment:
Please change to `players/v2` and so on...

datail: https://github.com/supota/player-api 's readme..